### PR TITLE
Add metadataUri to seller struct

### PR DESCRIPTION
**Pull request by anajuliabit**
This PR also reverts when no field was updated on seller and DR updates to avoid expending unnecessary gas.

### CODE REVIEW

File: contracts/protocol/facets/DisputeResolverHandlerFacet.sol [hunk 1]
```diff
@@ -289,6 +289,8 @@ contract DisputeResolverHandlerFacet is IBosonAccountEvents, ProtocolBase {
                 sender
             );
         }
+
+        require(updateApplied || needsApproval, NO_UPDATE_APPLIED);
```
Comment (mischat) [inline contracts/protocol/facets/DisputeResolverHandlerFacet.sol:293]: Not sure i understand this line, but otherwise this WIP looks good. Why is this needed, and how come it has been added to the DisputeResolverHandler, is this just so that update methods throw in a consistent way ?

**Review by mischat — COMMENTED** [review]
Otherwise this WIP looks good and should unblock the dapp / core components work

File: contracts/protocol/facets/DisputeResolverHandlerFacet.sol (hunk 1 above)
Comment (anajuliabit) [reply contracts/protocol/facets/DisputeResolverHandlerFacet.sol:293]: If no field has been updated or requested to be updated, there is no reason to execute the transaction. It is better to revert and return at least some gas to the user.

**Review by zajck — DISMISSED** [review]
LGTM

File: contracts/protocol/facets/DisputeResolverHandlerFacet.sol (hunk 1 above)
Comment (mischat) [reply contracts/protocol/facets/DisputeResolverHandlerFacet.sol:293]: Ok, got it.

**Review by mischat — APPROVED** [review]
Thanks for this ! Nice and speedy turn around ... we should cut v2.2.1 now

**Review by zajck — DISMISSED** [review]

**Review by zajck — APPROVED** [review]
